src/components/Request/index.js

```diff
@@ -0,0 +1,66 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import PropTypes from 'prop-types'
+import { Color, Space, Radius } from 'lib/theme'
+import { numericString } from 'lib/utils/strings'
+import { formatDate } from 'lib/utils/datetime'
+import { numberWithCommas } from 'lib/utils/number'
+import { SecondaryButton } from 'components/Button'
+import Text from 'components/Text'
+import { TEXT_TYPE } from 'components/Text/constants'
+import { styles } from './Request.styles'
+
+export const Request = ({ date, donation, id, name, onDelete, request }) => {
+  const numberDonation = numericString(donation)
+  const numberRequest = numericString(request)
```

Comment:
this function in the other project was used to strip out non-number characters. do the donations/requests contain anything besides ints?

it looks like it doesn't given that we are running `numberWithCommas` on it. if that's the case, you can use the unary operatory `+` before the number.